mill_local/__init__.py

```diff
@@ -22,81 +39,109 @@ def __init__(self, device_ip, websession, timeout=15):
-                        response.reason,
-                    )
-                return response.status
+    async def set_operation_mode_control_individually(self):
+        """Set operation mode to 'control individually'."""
+        return await self._set_operation_mode(OperationMode.CONTROL_INDIVIDUALLY)
+
+    async def set_operation_mode_off(self) -> None:
+        """Set operation mode to 'off'."""
+        await self._set_operation_mode(OperationMode.OFF)
 
     async def connect(self):
         """Get heater status."""
         return await self.get_status()
 
     async def get_status(self):
-        """Get heater status."""
+        """Get summary of the heater device information."""
         try:
-            self._status = await self._request("status")
+            self._status = await self._get_request("status")
         except (aiohttp.client_exceptions.ClientError, asyncio.TimeoutError):
             _LOGGER.error("Failed to get status", exc_info=True)
             return None
         return self._status
 
     async def fetch_heater_and_sensor_data(self):
-        """Get heater status."""
-        return await self._request("control-status")
+        """Get current heater state and control status."""
+        return await self._get_request("control-status")
+
+    async def _set_operation_mode(self, mode: OperationMode) -> None:
+        """Set heater operation mode."""
+        _LOGGER.debug("Setting operation mode to: '%s'", mode.value)
+        await self._post_request(command="operation-mode", payload={"mode": mode.value})
 
-    async def _request(self, command):
+    async def _post_request(self, command: str, payload: dict) -> None:
+        """HTTP POST request to Mill Local Api."""
+        with async_timeout.timeout(self._timeout):
+            async with self.websession.post(
+                    url=f"{self.url}/{command}",
+                    data=json.dumps(payload),
+            ) as response:
+                if response.status != 200:
+                    _LOGGER.error(
+                        "POST '%s' failed with response.status: %s and response.reason: %s",
+                        command,
+                        response.status,
+                        response.reason,
+                    )
+                    return
+
+                try:
+                    res = await response.json()
+                    if res["status"] != "ok":
+                        _LOGGER.error(
+                            "POST '%s' failed with result.status: %s, response.status: %s, response.reason: %s",
+                            command,
+                            res["status"],
+                            response.status,
+                            response.reason,
+                        )
+                        return
+                except aiohttp.client_exceptions.ClientError:
+                    _LOGGER.error("POST '%s' failed to parse json response", exc_info=True)
```

Comment:
```suggestion
                    _LOGGER.error("POST failed to parse json response", exc_info=True)
```

Reply:
Good catch, I forgot to add `command` as argument. Fixed in new commit!